explorer/ast/declaration.h

```diff
@@ -316,7 +316,7 @@ class MixinDeclaration : public Declaration {
                    std::vector<Nonnull<Declaration*>> members)
       : Declaration(AstNodeKind::MixinDeclaration, source_loc),
         name_(std::move(name)),
-        params_(std::move(params)),
+        params_(params),
```

Comment:
Hm, is it really worth `move`ing an `optional<T*>`? I doubt it, but maybe it's not worth fighting clang-tidy on this.